pkgs/os-specific/linux/ebtables/default.nix

```diff
@@ -12,16 +23,28 @@ stdenv.mkDerivation rec {
   makeFlags =
     [ "LIBDIR=$(out)/lib" "BINDIR=$(out)/sbin" "MANDIR=$(out)/share/man"
       "ETCDIR=$(out)/etc" "INITDIR=$(TMPDIR)" "SYSCONFIGDIR=$(out)/etc/sysconfig"
-      "LOCALSTATEDIR=/var"
+      "LOCALSTATEDIR=/var" "CC=${stdenv.cc}/bin/${stdenv.cc.targetPrefix}cc"
     ];
 
+  patches = lib.optional stdenv.hostPlatform.isMusl ./musl-compatibility.patch ++
+            lib.optional stdenv.hostPlatform.isAarch32 ./no-as-needed.patch;
+
   preBuild =
     ''
       substituteInPlace Makefile --replace '-o root -g root' ""
     '';
 
-  NIX_CFLAGS_COMPILE = "-Wno-error";
+  NIX_CFLAGS_COMPILE = lib.concatStringsSep " " (
+    ["-Wno-error"] ++
+    lib.optionals stdenv.hostPlatform.isMusl [
+      "-D__THROW="
+      "-Du_int8_t=uint8_t"
+      "-Du_int32_t=uint32_t"
+    ] ++
+    lib.optional enableDebug "-g3"
```

Comment:
We don't want this. This is something that you would set globally. You can use keepDebugInfo for this purpose:

https://github.com/NixOS/nixpkgs/blob/master/pkgs/stdenv/adapters.nix#L157-L165

Reply:
Er.. yeah, that makes sense. You can see my debugging process in this PR :)